DOCS-6433 edge metrics (#50)
mulesoft/docs-visualizer: modules/ROOT/pages/standalone-mule-setup.adoc

== Install Agent

To support Anypoint Visualizer for standalone Mule deployments, install the xref:monitoring::am-installing.adoc[Anypoint Monitoring agent].

> Anypoint Visualizer requires an additional header injection for standalone Mule deployments to appear properly with their correct connections.

Reviewer: ```suggestion
Anypoint Visualizer requires an additional header injection for standalone Mule deployments so that the deployments display the correct connections.
```

> Anypoint Visualizer requires an additional header injection for standalone Mule deployments so that the deployments display the correct connections.

Author: ```suggestion
Anypoint Visualizer requires header injection for standalone Mule deployments so that the deployments display the correct connections.
```

> Due to a potential performance impact, however, the header injection is disabled by default for Mule 4 apps. 

Reviewer: Hmm...this sounds kind of important. Perhaps the best way to treat a default disablement the must be enabled for Visualizer is by rewording a bit:

"Although header injection is disabled by default for Mule 4 apps, Anypoint Visualizer requires you to provide an additional header injection for standalone Mule deployments so that the deployments display the correct connections."
